CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,10 @@
-[cygnus-ngsi] [NGSIDynamoDBSink] [bug] Southeast regions are bad coded (#1448)
+[cygnus-ngsi][Docker] Add grouping rules files for multiagent (#1533)
```

Comment:
Changes in CHANGES_NEXT_RELEASE should show only the line corresponding to this PR. This seems to be a wrongly solved git merge conflict...

Reply:
It seems the CHANGES_NEXT_RELEASE stills having problems. An easy way to solve them is:

1. Remove CHANGES_NEXT_RELEASE file in your working copy for this branch
2. Download the one in master branch and replace it in your working copy for this branch
3. Edit the file to add:

```
 [cygnus-ngsi][NGSIEvent] Unable to deliver event: null pointer getAttributeForNaming (#1506) 
```

4. Commit the change and push the branch